audit/entry_formatter.go

```diff
@@ -95,6 +112,23 @@ func (f *EntryFormatter) Process(ctx context.Context, e *eventlogger.Event) (*ev
 		return nil, fmt.Errorf("%s: cannot audit event (%s) with no data: %w", op, a.Subtype, event.ErrInvalidParameter)
 	}
 
+	// Handle panics
+	defer func() {
+		r := recover()
+		if r == nil {
```

Comment:
This is a nit, but the following seems to be a much more common pattern, so it might make sense for us to align to that one
```
if r := recover(); r != nil {
   //logging, assigning erroring, etc
}
```

Reply:
That seems to be the case in the other ones I've seen, like this one in `audit_broker.go`, though maybe I'm totally misunderstanding what you mean :P :

```
		if r := recover(); r != nil {
			a.logger.Error("panic during logging", "request_path", in.Request.Path, "error", r, "stacktrace", string(debug.Stack()))
			retErr = multierror.Append(retErr, fmt.Errorf("panic generating audit log"))
		}
```

Reply:
Nope that's my bad. I totally missed that your original suggestion involved handling logging inside the if statement. Soz for the noise :D

Reply:
It's also on me for not posting an example in the initial comment, so no worries at all :D